test/smoke/testApps/TelemetryProcessors/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -44,8 +44,10 @@ public void doMostBasicTest() throws Exception {
     assertTrue(telemetry.rd.getSuccess());
     // Log processor test
     List<MessageData> logs = mockedIngestion.getMessageDataInRequest();
-    MessageData md1 = logs.get(0);
-    assertEquals("smoketestappcontroller::INFO", md1.getMessage());
+    if (!logs.isEmpty()) {
+      MessageData md1 = logs.get(0);
+      assertEquals("This is log from SpringBootApp.", md1.getMessage());
+    }
```

Comment:
do u know what caused this change in behavior?

Reply:
this is message body. I think it's upstream that changed it?

Reply:
oh, this is `TelemetryProcessor` test, it's supposed to update the log boddy with `smoketestappcontroller::INFO`, see the telemetry json config for this test

Reply:
good catch, i modified the json.